### ISSUE TITLE
custom rotation not working

### ISSUE THREAD

**Issue by X-SLAYER** [gauris26/flutter_animarker#18]
when I add a marker or RippleMarker with custom rotation while useRotation of Animaker is disabled it doesn't work

**Comment by CuongNV12** [comment]
I am also facing the same problem.

**Comment by leopi99** [comment]
I'm having the same issue, even if useRotation is set to true, the only "workaround" is to remove animarker but that's kind of pointless

**Comment by thetechguy2016** [comment]
I am also facing the same issue, did any one find how to rotate with Animarker, please let me know if you know.

**Comment by Nort432** [comment]
I still can't rotation the marker. How can i do it?

**Comment by gauris26** [comment]
Hi guys, I just submitted a beta version with this bugfix. Please try the prerelease and let me know.

**Comment by hdt786** [comment]
yes, @gauris26 I used the beta version and the problem has been resolved for custom rotation, but here I am facing the rotation animation 360 degree on my custom rotation degrees. So please at least help that how to disable animation on rotation only?
